server/controllers/stock/core.js

```diff
@@ -388,6 +388,7 @@ function getStockConsumption(periodIds) {
  * @description
  * Algorithm to calculate the CMM (consommation moyenne mensuelle) or average stock consumption
  * over a period for each stock item that has been consumed.
+ * NOTA: A FISCAL YEAR MUST BE DEFINED FOR THE FEATURE WORK PROPERLY
```

Comment:
This seems pretty important :)

Reply:
Yes... consumption is stored according period ids.